tools/analyse_query_usage.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+# Usage:
+#
+#   ./analyse_query_usage.sh my-file-of-urls.txt
+#
+#   cat my-file-of-urls.txt | ./analyse_query_usage.sh
+#
+#
+# For the input, a list of URLs, shows which are most common when you
+# ignore all query parameters. Eg, those that will be normalised in
+# mappings unless some query parameters are marked as important.
+#
+# Eg:
+#   162 http://www.planningportal.gov.uk/PpPortalSupport/ChangeLanguage
+#    45 http://www.planningportal.gov.uk/PpWeb/jsp/redirect.jsp
+#       ...
+#
+
+# Input is either file arg contents or stdin
+cat "$@" | \
+  # Take all of the URL leading up to `?`, or all if not present
+  cut -d? -f1 | \
```

Comment:
The output at this stage can include URLs which are identical apart from with/without trailing slashes, which would be removed by canonicalization. Could we drop trailing slashes to further condense the output to be more similar to canonicalization?